app/assets/javascripts/govuk_publishing_components/components/show-password.js

```diff
@@ -26,6 +26,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.showHide = document.createElement('button')
     this.showHide.className = 'gem-c-show-password__toggle'
     this.showHide.setAttribute('aria-controls', this.input.getAttribute('id'))
+    this.showHide.setAttribute('type', 'button')
```

Comment:
Worth including as part of the tests?

Reply:
Good point Andy, going to look at that today